frappe/public/js/frappe/list/list_sidebar.js

```diff
@@ -90,6 +83,15 @@ frappe.views.ListSidebar = class ListSidebar {
 			this.sidebar.find('.list-link[data-view="Image"]').removeClass('hide');
 			show_list_link = true;
 		}
+		// show map link if map_view doctype has get_coords or latitude and longitude
+		if ((JSON.stringify(frappe.listview_settings) !== '{}' &&
+            frappe.listview_settings[this.list_view.doctype].get_coords_method) ||
+            (this.list_view.meta.fields.find(i => i.fieldname === "latitude") &&
+            this.list_view.meta.fields.find(i => i.fieldname === "longitude")) ||
+			(this.list_view.meta.fields.find(i => i.fieldname === 'location') && this.list_view.meta.fields.find(i => i.fieldtype === 'Geolocation'))) {
+			this.sidebar.find('.list-link[data-view="Map"]').removeClass('hide');
+			show_list_link = true;
+		}
```

Comment:
@AminovE99 Why are there some many other modifications in this file ??

It sounds like you copy pasted file from v11 branch instead of just applying the needed updates, thus reverting this file 😕